### PR TITLE
refacto(ui): Add Breadcrumb Component from @centreon-ui

### PR DESCRIPTION
**Pull request by CoralieRubin**
# Pull Request Template

## Description

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:

Jsut move the component from a repo to another no visual change.

* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.
At the top of all the page you need to see the breadCrumb 

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by Thebarda** [comment]
You can remove `www/install` folder from commited files :)

**Comment by Thebarda** [comment]
As well, I think it is the time to move breadcrumb and link to TypeScript

File: www/front_src/src/components/breadcrumbWrapper/Link.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Link as RouterLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import { styled } from '@material-ui/core/styles';
+import Link from '@material-ui/core/Link';
```
Comment (Thebarda) [inline www/front_src/src/components/breadcrumbWrapper/Link.jsx]:
```suggestion
import { styled, Link } from '@material-ui/core';
```

File: www/front_src/src/components/breadcrumbWrapper/Breadcrumb.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Breadcrumbs from '@material-ui/core/Breadcrumbs';
```
Comment (Thebarda) [inline www/front_src/src/components/breadcrumbWrapper/Breadcrumb.jsx]:
```suggestion
import { makeStyles, Breadcrumbs} from '@material-ui/core';
```

**Comment by CoralieRubin** [comment]
I close this pr but i reopen one right the way with the good modification.

**Pull request closed without merging**